docker/runtime/backup-init.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+set -e
+
+BACKUP_PATH="/backup"
+
+# Make sure that required directories exist
+mkdir -p "${BACKUP_PATH}"
+mkdir -p "/etc/crontabs"
+chown git:git /backup
+chmod 2770 /backup
```

Comment:
Any reference to what does 2770 means?

Reply:
https://chmodcommand.com/chmod-2770/


> Chmod 2770 (chmod a+rwx,o-rwx,ug+s,+t,u-s,-t) sets permissions so that, (U)ser / owner can read, can write and can execute. (G)roup can read, can write and can execute. (O)thers can't read, can't write and can't execute.

The explanation for 2:
> Setuid and setgid (short for 'set user ID upon execution' and 'set group ID upon execution', respectively) are Unix access rights flags that allow users to run an executable with the permissions of the executable's owner or group respectively and to change behaviour in directories.


In my experience I found 2770 to work the best when I want to allow the owner user and group full control over their home directory, while I also want to fully restrict any other non privileged user from that space.